ui/app/AppLayouts/Profile/Sections/AboutContainer.qml

```diff
@@ -56,12 +56,46 @@ Item {
         font.weight: Font.Bold
         font.pixelSize: 14
     }
+    StyledText {
+        id: element13
+        text: "This software is licensed under under the <a href='https://github.com/status-im/nim-status-client/blob/master/LICENSE.md'>Mozilla Public License Version 2.0</a>."
```

Comment:
I don't know if "Mozilla Public License Version 2.0" is translatable. If it isn't, then you can do:
```qml
qsTr("This software is licensed under under the %1").arg("<a href='https://github.com/status-im/nim-status-client/blob/master/LICENSE.md'>Mozilla Public License Version 2.0</a>.")
```

If it is translatable, you can just split the string on multiple lines like so:
```qml
qsTr("This software is licensed under under the %1")
.arg(`<a href='https://github.com/status-im/nim-status-client/blob/master/LICENSE.md'>${qsTr("Mozilla Public License Version 2.0")}</a>.`)
```
It should work. I haven't tested though
The different lines are important, because we have 2 `qsTr`s and once transformed in `qsTrId` by the scripts, they need to each have a comment above them to give context to the translators.